src/server/twemcache/data/process.c

```diff
@@ -532,7 +571,7 @@ twemcache_process_read(struct buf **rbuf, struct buf **wbuf, void **data)
 
         status = parse_req(req, *rbuf);
         if (status == PARSE_EUNFIN) {
-            goto done;
+            return 0;
```

Comment:
I don't fully understand these changes of `return`ing instead of `goto`... `_cleanup` is no longer called, that seems to be on purpose, but I don't understand why